src/raftstore/store/worker/region.rs

```diff
@@ -162,14 +163,14 @@ impl PendingDeleteRanges {
         ranges
     }
 
+    fn remove(&mut self, start_key: &[u8]) -> Option<(u64, Vec<u8>, Vec<u8>)> {
```

Comment:
why not return PeerInfo directly? strange to return a tuple here.

Reply:
Because `insert` needs a tuple, so I think I should keep 2 interfaces similar.